image/docker_schema2.go

```diff
@@ -28,33 +28,33 @@ var gzippedEmptyLayer = []byte{
 // gzippedEmptyLayerDigest is a digest of gzippedEmptyLayer
 const gzippedEmptyLayerDigest = digest.Digest("sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4")
 
-type descriptor struct {
+type Descriptor struct {
```

Comment:
There are different descriptor schemas, and more importantly semantics / content expectations, in various formats (e.g. `OCI` expects different `MediaType` values). Name this perhaps `Schema2Descriptor`, please, to make it clear this is format-specific.